Try to fix build order bug

samu: job failed: /usr/local/bin/glib-compile-resources --sourcedir=common/gtk-3.0 --target=common/gtk-3.0/gtk-lighter.gresource common/gtk-3.0/gtk-lighter.gresource.xml
common/gtk-3.0/gtk-lighter.gresource.xml: Failed to locate checkbox-checked.png

--- common/gtk-3.0/meson.build.orig	2021-04-07 08:43:47 UTC
+++ common/gtk-3.0/meson.build
@@ -42,8 +42,10 @@ gtk3_asset_names = run_command(
 
 assets_svg = gtk3_ver / 'assets.svg'
 
+gtk3_assets = []
+gtk3_hidpi_assets = []
 foreach asset : gtk3_asset_names
-  gtk3_assets = custom_target(
+  gtk3_assets += custom_target(
     'gtk3-' + asset,
     input : assets_svg,
     output : asset + '.png',
@@ -58,7 +60,7 @@ foreach asset : gtk3_asset_names
     build_by_default : true
   )
 
-  gtk3_hidpi_assets = custom_target(
+  gtk3_hidpi_assets += custom_target(
     'gtk3-' + asset + '-hidpi',
     input : assets_svg,
     output : asset + '@2.png',
